test/e2e/opm_test.go

```diff
@@ -487,7 +487,7 @@ var _ = Describe("opm", func() {
 	}
 
 	Context("using docker", func() {
-		cmd := exec.Command("docker")
+		cmd := exec.Command("docker", "--version")
```

Comment:
Ended up just using `docker --version` and `podman --version` as the check for whether these tools are present locally (e.g. in $PATH) to avoid the skew between the two checks. For whatever reason, running `docker` locally without any arguments returns a zero return code, but logs the output to stderr in the e2e suite logs.